ui/src/app/components/apps-panel/apps-panel-button.tsx

```diff
@@ -52,28 +53,36 @@ interface Props {
   icon: IconProp;
   buttonText: string;
   disabled?: boolean;
+  disabledTooltip?: string;
 }
 export const AppsPanelButton = (props: Props) => {
-  const { disabled, onClick, icon, buttonText } = props;
+  const { disabled, onClick, icon, buttonText, disabledTooltip } = props;
+
+  const showTooltip = disabled && disabledTooltip;
+
   return (
-    <Clickable
-      {...{ disabled, onClick }}
-      style={{ padding: '0.5em' }}
-      data-test-id={`apps-panel-button-${buttonText}`}
-      propagateDataTestId
-    >
-      <FlexColumn
-        style={disabled ? buttonStyles.disabledButton : buttonStyles.button}
+    <TooltipTrigger content={disabledTooltip} disabled={!showTooltip}>
```

Comment:
The only change to this `return` statement is the addition of the `TooltipTrigger`.

We currently don't unit test `TooltipTrigger` since Enzyme has a hard time rendering `content`. I've manually tested this and provided screenshots.